kani-compiler/src/kani_middle/transform/body.rs

```diff
@@ -452,32 +436,19 @@ impl MutableBody {
     }
 }
 
+// TODO: Remove-me
```

Comment:
Can you expand on the TODO?

Reply:
sure... I thought it was obvious since it is an enum with one variant. :)

Reply:
😄  I confused this with another enum.

Reply:
```suggestion
// TODO: Remove this enum, since we now only support kani's assert.
```